uta_tools/data_sources/uta_database.py

```diff
@@ -0,0 +1,890 @@
+"""Module for UTA queries."""
```

Comment:
This module is pretty long -- if more stuff gets added later, it might be good to think about how to break its responsibilities up a bit